R/access.R

```diff
@@ -376,3 +376,62 @@ set_rights_and_access <- function(mn, pids, subject, permissions=c("read", "writ
 
   results
 }
+
+
+#' Check whether an object has public read access
+#'
+#' Check whether objects with given pids have public read access.
```

Comment:
This is super nitpicky, but if you could change this to "Check whether DataOne objects have been public read access set in their System Metadata."

Reply:
That's a much better description, thanks!

Reply:
Np, great work!  I'll merge this once travis finishes its build.